Proposed product descriptions/Reference Data Search Product Description.md

```diff
@@ -0,0 +1,126 @@
+
+2.  Biochemistry data:
+
+    We have developed new Solr schema to load reference biochemistry
+    data dictionaries for metabolic compounds, complexes, reactions and
+    roles, and subsystems. We propose to make the same data types
+    searchable using the reference data search interface.
+
+    We will also create data landing pages for each of the data types,
+    which will provide more information about the data and will be
+    linked from the search results.
+
+3.  Reference Data Solr Schema Enhancements:
+
+    We propose the following enhancements to the existing Solr schema to
+    provide better search experience to the users.
+
+-   **Search logic:** While searching for reference data using multiple
+    keywords, user AND as the default search logic, instead of using OR.
+    For example, currently if a user searches for *Escherichia coli HS*
+    using reference data search, 2339 genomes are returned, as they
+    match Escherichia OR coli OR HS. These represent too many false
+    positive matches and something not expected by the user. However, if
+    AND is used as the default search logic, it returns only two
+    matching genomes, which is exactly what user expected.
+
+-   **Genome metadata:** Genome metadata refers to the descriptive data
+    about a genome, which provide additional information about the
+    genomes, such as genome status, sequencing platform and depth,
+    isolation source, geographic location, host, collection date, other
+    clinical or environmental factors, phenotype related information,
+    and cross references to external databases, such as NCBI’s
+    BioSample, BioProject, Assembly, GenBank, and SRA database.
+
+    As there are hundreds (or sometimes even thousands) of genomes
+    sequenced for the same species and strain, metadata is absolutely
+    crucial to distinguish one genome from another and understand why
+    the genome was sequenced. Often, filtering the genome based on
+    available metadata is essential first step to identify a set of
+    genomes one wants to include in comparative genomic analysis. For
+    example, a user may want to select Brucella genomes isolated from
+    different hosts and compare them using pan-genome tool to understand
+    host-specific adaptations.
+
+-   **Value added annotations:** One of the objectives of the KBase
+    reference data ingestion process is to annotate all the microbial
+    genomes using a consistent vocabulary and also provide value added
+    annotations, such annotation of protein families, domains, and
+    operons, which provide rich source of information and form the basis
+    of many comparative genomic analysis methods. As we ad support for
+    value added annotations in the KBase genome annotation method, we
+    will also incorporate them into the object models, Solr schema, and
+    the reference data search.
+
+### Future work
+
+1.  Enhance data landing pages for genomes and genomic features.
+
```

Comment:
We should discuss how much of the value added analysis is in scope for Q2 in any case. Something to consider given we want these analyses ultimately to be in SDK format and easily extensible and there are issues with out to schedule and scale. Worth discussion. 

@dangunter it seems we have to discuss how to factor getting reference data into KBase, making it searchable and displaying the results... thus your comments about Landing pages. I agree anything done on landing pages should be specified. But is there a specific concern other than it might be biting off too much?